content/news/playtest-20160424.md

```diff
@@ -0,0 +1,69 @@
+
+We have now fixed all of those issues, including:
+
+* Crashes when opening the in-game menu on some maps (Fort Lonestar, for example).
+* Broken left click orders.
+* Way too many infantry units coming out of destroyed buildings.
+* Submarines not being targetable.
+* Stealth units being targetable even when stealthed under some circumstances.
+
+Additionally, the Red Alert mod has seen changes to its map pool: a few lower quality maps have been removed, while some new tournament-grade maps have been added.
+
+As usual, the complete list of changes can be seen [on our wiki](http://github.com/OpenRA/OpenRA/wiki/Changelog/HASHHASHASH).
+
+We promised in the last news post to detail the new map format and other map-related changes here. Skip to below the break for the full scoop.
+
+Installers for all supported operating systems are available from [the download page](/downloads). Of course we are still interested to hear about any bugs and issues you come across, so please don't hesitate to report them [on our bug tracker](http://bugs.openra.net), in the OpenRA [IRC channel](/community/), or here in the comments.
+
+The following note is copied from the previous news post, but we are going to repeat it here for accessibility.<br/>
+As many other open-source projects, we don't have any clear idea how much our software is actually used. There is a huge discrepancy between the number of downloads (about 50,000 currently for the latest release) and the number of online players (60-75 players concurrently at peak times). In order to get a better idea of this, and also to help with steering the future direction of the project, we decided to add an anonymous, opt-out data gathering facility to OpenRA. When enabled, the game will send us your operating system and .NET framework version, supported OpenGL version of your graphics card, your language settings, as well as the current mod. As already mentioned, this data is sent to us anonymously, using a randomly generated ID. If you are still feeling uncomfortable with this, you can disable this functionality by unchecking the "Send System Information" checkbox in the advanced settings dialog, or by starting the game using the `Debug.SendSystemInformation=false` argument.
+
+<div style="text-align:center" markdown="1">
+![Share of operating systems of our users](/images/news/20160424-os-shares.png)
+This is the share of operating systems among those players who used the playtest.
+</div>
+
+<hr />
+
+###Changes to map handling
+
+Custom maps are now stored in an additional sub-directory per mod version (e.g. `My Documents\OpenRA\maps\ra\playtest-20160403` on Windows).
+
+To ensure that outdated maps will not cause problems during gameplay, these maps will not be automatically moved when you upgrade the game. You will have to download updated versions from the [Resource Center](http://resource.openra.net) instead, or upgrade and move them yourself (see below).
```

Comment:
It would be good to mention that maps from playtest-20160403 can be copied or moved directly, and that only release-20151224 maps need to be upgraded.